--- mozc.orig/src/BUILD.ibus.bazel	2021-07-23 13:00:22.530416840 +0900
+++ mozc/src/BUILD.ibus.bazel	2021-07-23 13:02:26.840421949 +0900
@@ -17,7 +17,7 @@
         "include/libmount/**",
         "include/blkid/**",
         "include/glib-2.0/**",
-        "lib/x86_64-linux-gnu/glib-2.0/include/**",
+        "lib/glib-2.0/include/**",
     ]),
     copts = ["-pthread"],
     includes = [
@@ -25,7 +25,7 @@
         "include/libmount",
         "include/blkid",
         "include/glib-2.0",
-        "lib/x86_64-linux-gnu/glib-2.0/include",
+        "lib/glib-2.0/include",
     ],
     linkopts = [
         "-libus-1.0",
